### ISSUE TITLE
usergrid defaults to api.usergrid.com

### ISSUE THREAD

**Issue by ssvaidyanathan** [brandonscript/usergrid-nodejs#27]
Remove the defaulted api.usergrid.com or replace that with the new BaaS end point

**Comment by brandonscript** [comment]
Good catch. In the mean time you can override the baseUrl in the client init — see the readme for details.

**Comment by ssvaidyanathan** [comment]
Yeah - thats what I have suggested if you see my answer to the post in the community
